nri/index.html

```diff
@@ -1 +1,261 @@
+				<h3>NRIグループ国内・海外拠点一覧</h3>
+				<ul>
+					<li><a href="#">メインオフィス</a></li> 
+					<li><a href="#">グループカンパニー</a></li> 
+					<li><a href="#">その他の関連会社・団体</a></li> 	
+				</ul>
+			</div>
+
+			<div>
+				<h3>事業案内</h3>
+				<ul>
+					<li><a href="#">コンサルティング</a></li> 
+					<li><a href="#">金融ITソリューション</a></li> 
+					<li><a href="#">産業ITソリューション</a></li> 
+					<li><a href="#">IT基盤サービス</a></li> 
+				</ul>
+			</div>
+
+			<a href="#">&lt;前のページに戻る</a> 
+		</div><!-- /.sub-menu -->
+	</main>
+	<footer class="wrapper">
+		<div class="follow">
+			<ul>
+				<li>Follow Us</li>
+				<li><a href="#"><img src="./images/yt_icon_rgb.svg" alt="youtube"></a></li>
+				<li><a href="#"><img src="./images/icon-facebook.svg" alt="facebook"></a></li>
+				<li><a href="#"><img src="./images/icon-twitter.svg" alt="twitter"></a></li>
+				<li><a href="#"><img src="./images/icon-linkedin.svg" alt="linkedin"></a></li>
+			</ul>
+		</div><!-- /.follow -->
+	
+		<div class="sitemap">
+			<ul>
+				<li>
+					<ul>
+						<li><a href="#"><h4>ナレッジ・インサイト</h4></a></li>
+						<li><a href="#">刊行物</a></li>
+						<li><a href="#">書籍</a></li>
+						<li><a href="#">コラム</a></li>
+						<li><a href="#">セミナー資料</a></li>
+						<li><a href="#">レポート</a></li>
+						<li><a href="#">用語解説</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>ソリューション・サービス</h4></a></li>
+						<li><a href="#">マネジメントコンサルティング</a></li>
+						<li><a href="#">システムコンサルティング</a></li>
+						<li><a href="#">金融ITソリューション</a></li>
+						<li><a href="#">産業ITソリューション</a></li>
+						<li><a href="#">IT基盤サービス</a></li>
+						<li><a href="#">ソリューション・製品・サービス</a></li>
+						<li><a href="#">お客様事例</a></li>
+						<li><a href="#">未来創発センター</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>サステナビリティ</h4></a></li>
+						<li><a href="#">サステナビリティマネジメント</a></li>
+						<li><a href="#">取り組み</a></li>
+						<li><a href="#">ライブラリ</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>ニュース</h4></a></li>
+						<li><a href="#">ニュースリリース</a></li>
+						<li><a href="#">お知らせ</a></li>
+						<li><a href="#">イベント</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>企業情報</h4></a></li>
+						<li><a href="#">社長メッセージ</a></li>
+						<li><a href="#">経営方針</a></li>
+						<li><a href="#">NRIの強み</a></li>
+						<li><a href="#">企業案内</a></li>
+						<li><a href="#">国内・海外拠点</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>株主・投資家情報（IR）</h4></a></li>
+						<li><a href="#">IRニュース</a></li>
+						<li><a href="#">IRイベント</a></li>
+						<li><a href="#">経営方針</a></li>
+						<li><a href="#">財務・業績</a></li>
+						<li><a href="#">IRライブラリ</a></li>
+						<li><a href="#">株式・格付情報</a></li>
+						<li><a href="#">個人投資家の皆さまへ</a></li>
+					</ul>
+				</li>
+				<li>
+					<ul>
+						<li><a href="#"><h4>採用情報</h4></a></li>
+						<li><a href="#">メディア関係者の皆様へ</a></li>
+						<li><a href="#">ビジネスパートナーの皆様へ</a></li>
+					</ul>
```

Comment:
css課題の際に、無理やりに本番と同じデザインにすることもできますが、
文書構造的にも、cssの作りを考えても下記のようにしてしまうのが良いです。

`NRI JOURNAL`部分も同じ指摘です。

```suggestion
					<ul>
						<li><a href="#"><h4>採用情報</h4></a></li>
						<li><a href="#"><h4>メディア関係者の皆様へ</h4></a></li>
						<li><a href="#"><h4>ビジネスパートナーの皆様へ</h4></a></li>
					</ul>
```

![image](https://user-images.githubusercontent.com/1779811/115098171-6a602380-9f69-11eb-850a-58a270061e90.png)